src/scrapers/max.ts

```diff
@@ -291,6 +291,7 @@ class MaxScraper extends BaseScraperWithBrowser<ScraperSpecificCredentials> {
       },
       postAction: async () => redirectOrDialog(this.page),
       possibleResults: getPossibleLoginResults(this.page),
+      waitUntil: 'domcontentloaded' as 'domcontentloaded',
```

Comment:
Consider changing the signature to:
```diff
+ getLoginOptions(credentials: ScraperSpecificCredentials): LoginOptions {
- getLoginOptions(credentials: ScraperSpecificCredentials) {
```
then you can do:
```suggestion
      waitUntil: 'domcontentloaded',
```